src/app/server/ExampleAclStorage.h

```diff
@@ -0,0 +1,38 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app/server/AclStorage.h>
+
+namespace chip {
+namespace app {
+
+class ExampleAclStorage : public AclStorage
+{
+public:
+    /**
+     * Initialize must be called. It loads ACL entries for all fabrics (half-open range first to
```

Comment:
```suggestion
     * Initialize must be called. It loads ACL entries for all fabrics in the half-open range [first, last)
```

Reply:
Do the square brackets work with whatever Javadoc-like thing we're using, they won't confuse it? I thought about using them but thought it better to just write it in English (half-open range first to last, people can search what half-open means).

Reply:
The problem I was having is that "half-open" doesn't say which end is closed and which is open....

Reply:
Ah, I guess that's true, though it's conventional in C++ land that it's the way it's always about iterators. Honestly I should be able to put no docs there and it should be clear how it works. :-)

Reply:
I clarified how the half-open range works in the base function, then avoided repeating that in the derived function, and instead just explained what it actually does in the implementation.